proof/refine/ARM/CSpace_R.thy

```diff
@@ -2805,6 +2805,22 @@ lemma setCTE_arch [wp]:
   apply (wp|wpc|simp del: hoare_fail_any)+
   done
 
+(* FIXME arch-split: cleaner way of doing this? can't use obj_at_setObject2 on CTEs *)
+lemma setCTE_ko_at'_pde[wp]:
+  "setCTE p c \<lbrace> \<lambda>s. P (ko_at' (pde::ARM_H.pde) p' s) \<rbrace>"
```

Comment:
I don't see any immediate cleaner way. CTEs on Haskell have always been annoying, this proof still looks pretty Ok.